typescript/ws-protocol/README.md

```diff
@@ -1,13 +1,11 @@
-# Foxglove Studio WebSocket protocol
+# Foxglove WebSocket server and client
 
-This package provides server and client implementations of the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
-
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol) supports any data encoding, as long as it is supported by both client and server.
 
 ## Installation
 
 ```
-$ npm install @foxglove/ws-protocol
+$ npm install @foxglove/ws-protocol ws
```

Comment:
This is redundant with the section below, so maybe leave it off here, unless you wanted to combine them and delete the second one? I think the explanatory paragraph below is useful whether or not we have a separate line for `npm install ws`.